ui/packages/catalog-ui-search/src/main/webapp/react-component/location/gazetteer.js

```diff
@@ -197,7 +197,7 @@ class Gazetteer extends React.Component {
     })
   }
   isMultiPolygon(coordinates) {
-    return coordinates[0][0][0] !== null && coordinates[0][0][0][0] !== null
+    return coordinates[0][0][0] !== null && coordinates[0][0][0][0] !== undefined
```

Comment:
Should we also switch the `coordinates[0][0][0]` check to undefined?

Reply:
Yes, good catch. Addressed